api/disaggregated/v1/types.go

```diff
@@ -172,12 +172,20 @@ type CommonSpec struct {
 	//EnvVars is a slice of environment variables that are added to the pods, the default is empty.
 	EnvVars []corev1.EnvVar `json:"envVars,omitempty"`
 
+	//EnvFrom is a list of sources to populate environment variables in the container, the default is empty.
+	//+optional
+	EnvFrom []corev1.EnvFromSource `json:"envFrom,omitempty"`
+
 	//SystemInitialization for fe, be setting system parameters.
 	SystemInitialization *SystemInitialization `json:"systemInitialization,omitempty"`
 
 	// Multi Secret for pod.
 	// +optional
 	Secrets []Secret `json:"secrets,omitempty"`
+
+	// InitContainers is a list of containers that should be run before the app containers are started.
+	// +optional
+	InitContainers []corev1.Container `json:"initContainers,omitempty"`
```

Comment:
The 'InitContainers' defined in DDC are not passed into the actual 'PodTemplateSpec' construction